_docs/style/spacing.md

```diff
@@ -85,86 +110,70 @@ Exceptions are made for some elements such as dashboard panels and [cards]({{sit
 
 {% column left:"hxCol hxSpan-12" %}
 
-<!-- BUG: a[href="content"] apparently is a specifically styled thing in HelixUI... -->
+## Content spacing
+
+Content is any information displayed inside a container. Ensure that you conform to line height and bottom margin spacing values when you vertically stack content components. Bottom margins contain extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy.
 
-## Content text
+There are two types of content:
 
-Content lives inside the container. Content is stacked vertically using line-heights and bottom margins (depending on type of content). Bottom margins add extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy. Content contains:
+- **Headers**: Headers are the highest level of the content hierarchy and are used for page titles, section titles, sub-section titles, and  container titles.
+- **Body elements**: Body elements are located just below the header and consist of list items, drop-down selectors, and paragraphs.
 
-- Headers are at the highest level of hierarchy for content (page title, section title, sub-section title, container title)
-- Just below the header hierarchy is the data interspersed in the form of paragraphs, lists, forms, and tables that we will refer to as leaf-nodes.
+Apply a *stack* value to the space below a [header](#headers) (8px) and [body elements](#body-elements) (4, 8, or 16px according to the content type).
 
 {% endcolumn %}
 
 </div>
 
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-**Baseline grid is multiples of four.**
-Just like using columns as guides in your design, the baseline grid is a way to help you make decisions and build consistency into your layout based on multiples of four.
+**Baseline value**: The Helix spacing system uses four as the baseline number on which all other spatial values are calculated. For example, the font size value for all headings are multiples of four. The baseline value helps you build consistency into your layout.
```

Comment:
the text "four" confused me at first. can we use the number instead? or in addition?

Reply:
sure, switched to 4.